subtrees/z_quantum_actions/Makefile

```diff
@@ -52,19 +52,19 @@ flake8-default: clean
 
 mypy-default: clean
 	@echo scanning files with mypy: Please be patient....
-	@mypy --ignore-missing-imports --namespace-packages src
+	@mypy src tests
 
 black-default: clean
 	@black --check src tests
 
 isort-default: clean
-	@isort --check --profile black src tests
+	@isort --check src tests
```

Comment:
When did we remove black setting from isort?

Reply:
The black setting is now specified in the pyproject.toml provided by z-quantum-actions. Not sure when it was added to z-quantum-actions, but this PR is what configured qe-forest to use pyproject.toml